features/admin-file-attributes.feature

```diff
@@ -0,0 +1,132 @@
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+
+  @ok @published
+  Scenario: See Attribute, Edit, Delete, and Save buttons on admin file update page
+    Given I sign in as an admin
+    When I am on "/adminFile/update/id/13973"
+    Then I should see a button "New Attribute"
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    And I should see a button input "Edit"
+    And I should see a button input "Delete"
+    And I should see a button input "Save"
+
+  @ok @javascript @published
+  Scenario:  See the File Attribute Value of the last modified attribute on admin file view page
+    Given I sign in as an admin
+    When I go to "/adminFile/view/id/13973"
+    Then I should see field "File Attribute Value" with "2013-7-15 "
+
+
+  @ok @javascript @published
+  Scenario: Delete a last modified attribute on admin file update page
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/13973"
+    When I press "Delete"
+    Then I should not see "last_modified"
+    And I should not see "2013-7-15"
+    And I should not see a button "Delete"
+
+  @ok @javascript @published
+  Scenario: File Attribute value is empty empty after deleting an attribute and saving
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/13973"
+    When I press "Delete"
+    And I press "Save"
+    Then I am on "/adminFile/view/id/13973"
+    And I should not see "File Attribute Value"
+
+  @ok @javascript @published
+  Scenario: File attribute deletion is recorded in History tab
+    Given I sign in as an admin
+    And I go to "/adminFile/update/id/13973"
+    When I press "Delete"
+    And I press "Save"
+    Then I go to "/dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    And I should see "History" tab with text "Termitomyces_assembly_v1.0.fa.gz: file attribute deleted"
+
+  @ok @javascript @nonPublished
+  Scenario: See a keyword attribute and a camera parameters attribute on admin file update page
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/95354"
+    And I should see a file attribute table
+    | Attribute Name    | Value         | Unit |
+    | keyword           | test Bauhinia |      |
+    | camera parameters | test photo    |      |
+
+  @ok @javascript @nonPublished
+  Scenario: See File Attribute Value on admin file view page
+    Given I sign in as an admin
+    When I go to "/adminFile/view/id/95354"
+    Then I should see field "File Attribute Value" with "test Bauhinia test photo "
```

Comment:
`test Bauhinia test photo` - these are 2 file attribute values joined together? If so, I think it's better to split them into 2 separate file attributes otherwise it looks like it is a single file attribute with a value `test Bauhinia test photo`.

Reply:
Now, the file attribute value will be separated by `,` like this `test Bauhinia, test photo`